packages/SwingSet/test/test-liveslots.js

```diff
@@ -1005,6 +1005,7 @@ test('dropImports', async t => {
     () => {
       return { buildRootObject: build };
     },
+    false,
```

Comment:
part of me wants to suggest that the `makeLiveSlots()` argument should declare a default `allowFakeDurable = false`, and then this line goes away, but part of me thinks explicit is better

Reply:
Suspenders AND belt: I'll default the argument in `makeLiveSlots` but pass it explicitly here anyway.